### PR TITLE
Update deployment images to v1.1.0

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
/kind bug

**What this PR does / why we need it**:
This PR updates deployment images to v1.1.0.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xing-yang** [comment]
@msau42 @j-griffith PTAL.

File: deploy/kubernetes/setup-csi-snapshotter.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ spec:
       serviceAccount: csi-snapshotter
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.0.1
+          image: quay.io/k8scsi/csi-provisioner:v1.1.0
```
Comment (msau42) [inline deploy/kubernetes/setup-csi-snapshotter.yaml]: should the examples here just use canary so we don't have to remember to keep updating it every release?

Comment (xing-yang) [reply deploy/kubernetes/setup-csi-snapshotter.yaml]: Canary images may not work with each other before they are released. For example, I may have modified external-snapshotter APIs in the master but have not got time to update external-provisioner code yet, so the canary images won't work with each other.

**Pull request merged**